src/components/organisms/UiContentArea/_internal/UiContentAreaItem.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <slot v-if="isActive" />
+</template>
+
+<script setup lang="ts">
+import {
+  computed,
+  inject,
+  onBeforeUnmount,
+} from 'vue';
+import type { Ref } from 'vue';
+import type { ContentAreaItem } from '../UiContentArea.vue';
+
+const props = defineProps({
+  /**
+   * Use this props to set inside pages item label.
+   */
+  label: {
+    type: String,
+    default: '',
+  },
+  /**
+   * Use this props to set inside pages item title.
+   */
+  title: {
+    type: String,
+    default: '',
+  },
+  /**
+   * Use this props to set inside pages item name.
+   */
+  name: {
+    type: String,
+    default: '',
+  },
+});
+
+const activeItemName = inject('activeItemName') as Ref<string>;
+const isActive = computed(() => activeItemName.value === props.name);
+
+const item = computed(() => ({
+  label: props.label,
+  title: props.title,
+  name: props.name,
+}));
+const items = inject('items') as Ref<Record<string, ContentAreaItem>>;
+items.value[props.name] = item.value;
```

Comment:
Why does it happen in the global scope of the script? 🤔 🙂

Reply:
global scope? It's use `itemsToHandle` in UiHorizontalPaging. It's required to support adding items by default slot. Item add it to `itemsToHandle` and it will be displayed in menu